app/views/shared/_analytics_table.html.erb

```diff
@@ -9,44 +9,40 @@
         <col class="table-divider">
         <col>
         <col>
-        <col class="table-divider">
+        <col>
         <col>
         <col>
         <col class="table-divider">
         <col>
         <col>
+        <col>
+        <col>
+        <col>
+        <col class="table-divider">
```

Comment:
Is this one extra? We should have 3 new cols right?

Reply:
Thanks for catching!